### DISCUSSION TITLE
"spreadsheet" style editing

### DISCUSSION THREAD

**Discussion by jcollie** [netbox-community/netbox#10380] [Ideas]
In the near future, we may be updating our NetBox documentation with all of the physical labels on all of our patch panels. It would be nice if there was a way to edit the labels without opening up an editing page for every port. For example, if you go to this patch panel on the demo site, you'd have to click edit on all 48 ports to change the labels. The bulk edit isn't an option because that would set all of the labels to the same value. If there was a way to edit some of the properties like it was a spreadsheet it would really speed things up. I would be fine with only being able to edit basic things like the label, description, type, color, etc. 

https://demo.netbox.dev/dcim/devices/74/front-ports/

**Comment by candlerb** [comment]
Some sort sort of "update from CSV" would be nice.

In the short term, writing a Custom Script to do specifically what you need might be the best option. It can accept a file upload via [FileVar](https://docs.netbox.dev/en/stable/customization/custom-scripts/#filevar), or a [TextVar](https://docs.netbox.dev/en/stable/customization/custom-scripts/#textvar) text area.

**Reply by jeremystretch** [reply]
> Some sort sort of "update from CSV" would be nice.

#7961 has been opened for this, and we're hoping to get it into v3.4.
